.github/workflows/publish.yaml

```diff
@@ -98,3 +106,9 @@ jobs:
           build-args: |
             GIT_BRANCH: ${{ github.ref_name }}
             GIT_COMMIT: ${{ github.sha }}
+
+      - name: Sign the images with GitHub OIDC Token
+        run: cosign sign ${TAGS}
+        env:
+          TAGS: ${{ steps.meta.outputs.tags }}
```

Comment:
It is better practice to sign the digest here, e.g. https://github.com/actions/starter-workflows/blob/3603e1f7d1629d3e9b99a7fd2e6ad72e42e8e84e/ci/docker-publish.yml#L93

Reply:
True and also possible. Only reason i hasn't is this seemed a smaller change but i can add trivially.

Reply:
Should be good to go now, but happy to hear your thoughts. If fine, mark as resolved.